### ISSUE TITLE
Linked XML metadata

### ISSUE THREAD

**Issue by federicomorando** [nexacenter/public-contracts#9]
We should publish (and link to the contracting authority) the JSON with the URL of the original XML file, info about whether it validates or not, date of all our successful downloads and related links, license, etc.
Of course, all in RDF

**Comment by nicorusti** [comment]
We had discussed that issue previously. 
For each contract, we already included in the RDF the link of the xml file where the contract is present. 
We took this information from the xml file itself, and sometimes it may be wrong. This information has to be replaced with the real link where we downloaded the dataset (present in downloadInfo.json). For this purpose, the code for generation of the json files has to be modified. DownloadInfo.json  also contains the date of last dataset download, and wether a dataset contains valid data. 
Furthermore, the stats about the n. of parseable datasets is present in downloadStats.json, and it would make much more sense to have all the stats aggregated in the stats.json (which is accessible, for each PA, through public-contract.nexacenter.org).

**Comment by federicomorando** [comment]
OK, great. I think it would also be useful to have a property linking each PA to the info concerning its XML file, including URL, license, availability, etc. All that with appropriate information about the moment in which we checked (apart from the license, which should be the one of the XML we processed). I think we should also keep a copy of all the XML files we download (including obsolete ones, for the records).
